### PR TITLE
Tweaks lights to spark less

### PR DESCRIPTION
**Pull request by VileBeggar**
## About The Pull Request
Makes lights spark less.

## Why It's Good For The Game
These things spark way too often, and it's frankly quite annoying, it's like the bulbs are made out of wet toilet paper or something.

## Changelog
:cl: Vile Beggar
tweak: Lights now spark less.
/:cl:

### CODE REVIEW

**Review by 81Denton — DISMISSED** [review]

File: code/modules/power/lighting.dm [hunk 1]
```diff
@@ -9,8 +9,8 @@
 #define LIGHT_BROKEN 2
 #define LIGHT_BURNED 3
 
-#define BROKEN_SPARKS_MIN 15 SECONDS
-#define BROKEN_SPARKS_MAX 30 SECONDS
+#define BROKEN_SPARKS_MIN 30 SECONDS
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/power/lighting.dm]:
```suggestion
#define BROKEN_SPARKS_MIN (30 SECONDS)
```

File: code/modules/power/lighting.dm [hunk 2]
```diff
@@ -9,8 +9,8 @@
...
+#define BROKEN_SPARKS_MAX 90 SECONDS
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/power/lighting.dm]:
```suggestion
#define BROKEN_SPARKS_MAX (90 SECONDS)
```

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
good pr

**Review by ExcessiveUseOfCobblestone — APPROVED** [review]

**Review by Cyberboss — APPROVED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
should have claimed it was causing lag so I could early merge tbh lol

**Comment by VileBeggar** [comment]
i'm a good man i never lie

**Pull request merged**